src/components/navigation-bar/navigation-bar.style.js

```diff
@@ -36,6 +37,13 @@ const StyledNavigationBar = styled.nav`
     margin-right: 10px;
   }
 
+  ${({ stickyPosition, stickyOffset }) =>
+    STICKY_POSITIONS.includes(stickyPosition) &&
```

Comment:
This check isn't necessary because the `stickyPosition` prop can only be set to `top` or `bottom` anyway

Reply:
thanks, changed